packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageTypes.java

```diff
@@ -0,0 +1,27 @@
+package expo.modules.image;
+
+import android.widget.ImageView;
+import android.widget.ImageView.ScaleType;
+
+import com.facebook.react.bridge.JSApplicationIllegalArgumentException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ExpoImageTypes {
```

Comment:
When implementing `expo-notifications` I had to add several enum types to the package and I came up with this _enum pattern_, where for every JS-native enum we define an enum + a couple of helper functions. This has some advantages:
- an instance of an enum holds information about both JS and native value (coherency)
- putting different enums into different classes doesn't add too much bloat to one file
- with colocated functions we can convert back and forth between values of different realms
- you can use anything as native type (Strings too)
- an extra layer of abstraction between JS enum and native values (not exporting `ScaleType.FIT_CENTER` directly to JS)

Disadvantages of current implementations is `o(n)` searching when converting (could be mitigated with a simple hashmap creation).

I haven't tried to push to make this a convention in this repository, but so far I like it. What do you think about this pattern? An example would be:

https://github.com/expo/expo/blob/d09fc44c62ae96e0347e1909a774b1b93d866b49/packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/enums/AudioContentType.java

Reply:
Hey! I like this approach and the effort in standardizing this kind of thing.  I've added a dedicated `ExpoImageResizeMode` enum, please let me know whether this is what you had in mind.

While making the change, I came across this bug:
- There seems to be an error in the AudioContentType example that you provided. This line should use `getNativeValue`: https://github.com/expo/expo/blob/d09fc44c62ae96e0347e1909a774b1b93d866b49/packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/enums/AudioContentType.java#L39